packages/angular_devkit/build_angular/src/dev-server/index.ts

```diff
@@ -89,12 +90,12 @@ export function serveWebpackBrowser(
   async function setup(): Promise<{
     browserOptions: json.JsonObject & BrowserBuilderSchema;
     webpackConfig: webpack.Configuration;
-    webpackDevServerConfig: WebpackDevServer.Configuration;
     projectRoot: string;
     locale: string | undefined;
   }> {
     // Get the browser configuration from the target name.
     const rawBrowserOptions = await context.getTargetOptions(browserTarget);
+    options.port = await checkPort(options.port || 4200, options.host || 'localhost');
```

Comment:
`0` is a valid value for port but is also falsy so `||` can't be used.

Reply:
That is correct, though that is the current behaviour. I’ll update to use `??`